runtime/testy/Ansible.py

```diff
@@ -222,16 +228,101 @@ def start(self):
 
         try:
             while True:
-                nextCall = time.time()
                 event = sel.select()
```

Comment:
This is a receiver, not a sender. Throttling belongs in sender (in this case, Dawn), not the receiver.

If you want to handle receiving packets faster than this thread can handle, the correct was to implement this is by dropping packets.